### PR TITLE
Styled, added JS function.

### PR DESCRIPTION
**Pull request by evanmorrisdev**
Changed style of page using css file.
- going to add more to the page once I have completed other project.

Added function to return the keycode information.
- linked up and should run when opening the webpage. (tested).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -8,6 +8,13 @@ html, body {
     font-family: 'Fira Sans', sans-serif;
 }
 
+.pageDisplay {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+
```
Comment (tamjankowska) [inline style.css:17]: You have a bit of whitespace here.

File: app.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ const whichEventResult = document.getElementById("whichResult");
```
Comment (tamjankowska) [inline app.js:5]: More whitespace!

File: app.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ const whichEventResult = document.getElementById("whichResult");
 
 
 
-headerResult.addEventListener("keydown", (event) => {
-    
+document.addEventListener("keydown", (event) => {
+    headerResult.textContent = event.which;
+    keycodeResult.textContent = event.code;
```
Comment (tamjankowska) [inline app.js:10]: Nice :)

**Review by tamjankowska — CHANGES_REQUESTED** [review]
All looks good, I would just advise tidying up your whitespace.

**Comment by evanmorrisdev** [comment]
removed whitespace as advised.

**Comment by tamjankowska** [comment]
Thanks Evan! Looks good :)

**Pull request merged**